### ISSUE TITLE
Backpropagation

### ISSUE THREAD

**Issue by shidilrzf** [amber0309/HSIC#6]
Thanks for the code. Is this loss differentiable? I want to know if I can add it to my objective function.

**Comment by amber0309** [comment]
Hi 

This code is in numpy so not differentiable. You can reimplement a differentiable version in PyTorch or TensorFlow.
